### DISCUSSION TITLE
Is there a way to determine what version of handlebars is being used?

### DISCUSSION THREAD

**Discussion by eric-g-97477** [handlebars-lang/handlebars.js#1828] [Q&A]
For example, in the console, I can enter Ember.VERSION and get the version of Ember JS that is being used.

Is there something similar for handlebars? How can I determine what version is being used?

**Comment by jaylinski** [comment]
Yes, you can write `Handlebars.VERSION`.

https://github.com/handlebars-lang/handlebars.js/blob/a9a8e403213583ca90cb7c872d3a22796c37d961/lib/handlebars/base.js#L8-L10
